encrypt/ibe/ibe.go

```diff
@@ -9,10 +9,10 @@ import (
 	"errors"
 	"fmt"
 
-	"go.dedis.ch/kyber/v3"
-	"go.dedis.ch/kyber/v3/group/mod"
-	"go.dedis.ch/kyber/v3/pairing"
-	"go.dedis.ch/kyber/v3/util/random"
+	"go.dedis.ch/kyber/v4"
```

Comment:
I'm guessing there's a reason this is all commented out, but it seems suspicious - could just nuke it